src/bcf_clean_overpar.cpp

```diff
@@ -358,7 +340,7 @@ List bcfoverparRcppClean(NumericVector y_, NumericVector z_, NumericVector w_,
 
     if(verbose_sigma){
         if(iIter%status_interval==0) {
-            Rcout << "iteration: " << iIter << " sigma: "<< sigma << endl;
+            Rcout << "iteration: " << iIter << " sigma/SD(y): "<< sigma << endl;
```

Comment:
does SD(y) need to be different in the weighted context? (I know this is just for printing progress)